### PR TITLE
Login email domain whitelist has updated

### PR DESCRIPTION
**Pull request by kmtama**
### Refactored Email Domains List

Some domain names have duplicate entries. Removed the duplication. The email Whitelist is up to date with the required ones.

### Trello Card link
https://trello.com/c/3B799i9B/2-email-domain-status

### CODE REVIEW

**Comment by naanambiah** [comment]
Hello,
How do I sign in?
Nana

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Review by ShocKwav3 — CHANGES_REQUESTED** [review]

File: src/emailDomains.js [hunk 1]
```diff
@@ -11,7 +11,6 @@ const domainObject =
      'helsinki.fi',
      'aalto.fi',
      'jyu.fi',
-     'tuni.fi',
```
Comment (ShocKwav3) [inline src/emailDomains.js:14]: ohfsduihg

File: src/services/user.js [hunk 2]
```diff
@@ -171,6 +171,7 @@ const createUser = async (req, res, next) =>
             if (email.endsWith(domain))
             {
                 domainFlag = true;
+                console.log(domainFlag);
```
[the remark this answers is not in the document]
Comment (kmtama) [reply src/services/user.js]: Have revert changes. Please review now

File: src/services/user.js [hunk 3]
```diff
@@ -226,6 +226,7 @@ const createUser = async (req, res, next) =>
             languagesToTeach: languagesToTeach,
             languagesToLearn: languagesToLearn,
             userIsActivie: userIsActivie,
+            isActivated: false,
```
Comment (ShocKwav3) [inline src/services/user.js:229]: This is still shown as a change :)

**Review by ShocKwav3 — CHANGES_REQUESTED** [review]

**Comment by ShocKwav3** [comment]
Please update the description with the trello card link

File: src/services/user.js (hunk 3 above)
Comment (kmtama) [reply src/services/user.js:229]: for local, it was true to run the program which I firstly commit by mistake. Now I have correct it with another commit that's why it is showing as a change.

Comment (ShocKwav3) [reply src/services/user.js:229]: But the point is, if it shows then this is different than your target branch. You can check this file on your local by checking out to devel branch or using the github file explorer ui.

Comment (ShocKwav3) [reply src/services/user.js:229]: https://github.com/Tandem-App-Tuni/Kivako-app-backend/blob/devel/src/services/user.js#L228

The key `isActivated` does not exist. You have added it. Why?

**Review by Riikka-O — APPROVED** [review]

**Review by ShocKwav3 — APPROVED** [review]

**Review by juusohuhtivuo — APPROVED** [review]

**Review by janakavishwajith — APPROVED** [review]

File: src/services/user.js [hunk 4]
```diff
@@ -1,776 +0,0 @@
-/********
- * user.js file (services/users)
```
[the remark this answers is not in the document]
Comment (kmtama) [reply src/services/user.js]: No, it was deleted by mistake. I have reverted it and only remove the changes that I do locally. Now this file will not have any conflict with the original one.

**Review by tamarameghla — APPROVED** [review]
Ok

**Pull request merged**